test/integration/scipy_test.py

```diff
@@ -0,0 +1,138 @@
+# skip-if: True
+# This test will take a long time(51mins).
+# 3100.61s user 490.14s system 118% cpu 50:30.63 total
+# So please test it manually.
+
+import os
+import sys
+import subprocess
+sys.path.append(os.path.dirname(__file__) + "/../lib")
+from test_helper import run_test
+
+"""
+Using this test file.
+
+SciPy C source code files generated individually by Cython, it may contains
+implicit declaration of functions when compiling. Pyston set the flag
+"-Werror=implicit-function-declaration". It will trigger errors like:
+"error: implicit declaration of function 'zcopy_' " when building SciPy.
+This option was disabled in CPython. But enabled in Pyston. So if you want build
+and test SciPy in Pyston, please revert the second change in
+https://github.com/dropbox/pyston/commit/949dfeb1f675ef05ddfd04333e7e6c0f9887860c
```

Comment:
Hmm that doesn't sound like that would be the issue then :/

I looked into it a bit, and it looks like there are quite a lot of implicit-function-declaration warnings in SuperLU, to the point that I don't think it makes sense for us to try to fix up the scipy code.

This is hacky, but maybe we could do something like this?
```
if not any ('scipy' in s for s in sys.argv):
    cc_args = cc_args + ["-Werror=implicit-function-declaration"]
```